usaspending_api/common/experimental_api_flags.py

```diff
@@ -17,4 +25,46 @@ def is_experimental_elasticsearch_api(request: Request) -> bool:
     """
     Returns True or False depending on if the expected_header_value matches what is sent with the request.
     """
-    return request.META.get(EXPERIMENTAL_API_HEADER) == ELASTICSEARCH_HEADER_VALUE
+    use_es = request.META.get(EXPERIMENTAL_API_HEADER) == ELASTICSEARCH_HEADER_VALUE
+    if not use_es:
+        mirror_request_to_elasticsearch(request)
```

Comment:
This is an `is` function, which implies it can be called as many times as desired without side-effects. There might be implementations where code checks for being elasticsearch in two places, making two forked requests.

Reply:
Fair point. Can you find any?

Reply:
I can't see into the future